src/ensembl/src/content/app/custom-download/containers/pre-filters-panel/PreFilterPanel.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FunctionComponent } from 'react';
+import { connect } from 'react-redux';
+import RoundButton, {
+  RoundButtonStatus
+} from 'src/shared/round-button/RoundButton';
+
+import { getSelectedPreFilter } from '../../state/customDownloadSelectors';
+import {
+  updateSelectedPreFilter,
+  togglePreFiltersPanel
+} from '../../state/customDownloadActions';
+
+import { RootState } from 'src/store';
+
+import styles from './PreFilterPanel.scss';
+
+type PreFilterPanelProps = StateProps & DispatchProps;
+
+const PreFilterPanel: FunctionComponent<PreFilterPanelProps> = (
+  props: PreFilterPanelProps
+) => {
+  const filterOnClick = (filter: string) => {
+    props.updateSelectedPreFilter(filter);
+    props.togglePreFiltersPanel(false);
+  };
+
+  return (
+    <section className={styles.preFilterPanel}>
+      <div className={styles.panelTitle}>Select pre-filters</div>
+      <div className={styles.panelContent}>
+        <div className={styles.filterButtons}>
+          <RoundButton
+            onClick={() => {
+              filterOnClick('Genes/Transcripts');
+            }}
+            status={RoundButtonStatus.INACTIVE}
+            classNames={styles}
+          >
+            Genes/Transcripts
+          </RoundButton>
+          <RoundButton
+            onClick={() => {
+              filterOnClick('Variation');
+            }}
+            status={
+              RoundButtonStatus.DISABLED
+              // props.selectedPreFilter === 'Variation'
+              //   ? RoundButtonStatus.ACTIVE
+              //   : RoundButtonStatus.INACTIVE
```

Comment:
Is the above commented code relevant?

Reply:
I've temporarily disabled it as we don't support these filters yet.